http/Cargo.toml

```diff
@@ -6,26 +6,25 @@ edition = "2018"
 build = "build.rs"
 
 [build-dependencies]
-vergen = "3.0.4"
 prost-build = "0.6.1"
+vergen = "3.1.0"
 
 [dependencies]
-log = "0.4"
-env_logger = "*"
+base64 = "0.12.0"
```

Comment:
Do we need both base64 and multibase (it has `multibase::Base::Base64Pad` for what base64 generally means). I used the latter in pubsub (all fields needed to be base64).

Reply:
It's not used at all, since the api expects base58 but looks like dag-json which mandates base64